### PR TITLE
Implement autodiscovery v2 redirect to v1

### PR DESCRIPTION
**Pull request by JustasUmbrasas**

### CODE REVIEW

File: automx2/views/autodiscoverv2.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class OutlookView(MethodView):
+    """Autoconfigure mail, Outlook-style."""
+
+    def get(self):
+        if request.args.get('Protocol') == "autodiscoverv1":
+            value = {
+                "Protocol": "AutodiscoverV1",
+                "Url": "https://autodiscover-dev.hostinger.io/autodiscover/autodiscover.xml"
```
Comment (ledgr) [inline automx2/views/autodiscoverv2.py]: hardcoded url?

File: automx2/views/autodiscoverv2.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+from flask import request
+from flask.views import MethodView
+
+
+class OutlookView(MethodView):
+    """Autoconfigure mail, Outlook-style."""
+
+    def get(self):
+        if request.args.get('Protocol') == "autodiscoverv1":
```
Comment (ledgr) [inline automx2/views/autodiscoverv2.py:32]: File is `autodiscoverv2.py`, but defines `"Protocol": "AutodiscoverV1"`?

File: automx2/views/autodiscoverv2.py (hunk 1 above)
Comment (JustasUmbrasas) [reply automx2/views/autodiscoverv2.py]: not anymore

File: automx2/views/autodiscoverv2.py (hunk 2 above)
Comment (JustasUmbrasas) [reply automx2/views/autodiscoverv2.py:32]: `autodiscover.json` -> is autodiscoverv2 implementation. 

This only does a redirect from `autodiscover.json` to `autodiscover.xml` if autodiscoverv1 protocol is requested.

Comment (JustasUmbrasas) [reply automx2/views/autodiscoverv2.py:32]:
```
curl 'https://domain.tld/autodiscover/autodiscover.json?Email=user%40domain.tld&Protocol=autodiscoverv1&RedirectCount=1'
{"Protocol": "AutodiscoverV1", "Url": "https://domain.tld/autodiscover/autodiscover.xml"}
```

**Comment by JustasUmbrasas** [comment]
Closing for now as not fully supporting AutodiscoveryV2 brings confusion between email clients.

**Pull request closed without merging**